lib/capybara/node/finders.rb

```diff
@@ -48,21 +54,19 @@ def find(*args, **options, &optional_filter_block)
 
       ##
       #
-      # Find an {Capybara::Node::Element} based on the given arguments that is also an ancestor of the element called on. +ancestor+ will raise an error if the element
-      # is not found.
+      # Find an {Capybara::Node::Element} based on the given arguments that is also an ancestor of the element called on.
+      # {#ancestor} will raise an error if the element is not found.
       #
-      # +ancestor+ takes the same options as +find+.
+      # {#ancestor} takes the same options as {#find}.
       #
       #     element.ancestor('#foo').find('.bar')
       #     element.ancestor(:xpath, './/div[contains(., "bar")]')
       #     element.ancestor('ul', text: 'Quox').click_link('Delete')
       #
-      # @param (see Capybara::Node::Finders#find)
+      # @param (see #find)
       #
       # @macro waiting_behavior
       #
-      # @option options [Symbol] match        The matching strategy to use.
-      #
```

Comment:
[note] I think the `match` doc is unnecessary because it is already described in `#find`.

https://github.com/teamcapybara/capybara/pull/2204/files#diff-e0a7e0b5db6bbb0355a03b1e6ee1e3b0R45